rest-villains/src/main/resources/application.properties

```diff
@@ -12,6 +12,8 @@ quarkus.hibernate-orm.database.generation=drop-and-create
 villain.level.multiplier=0.5
 %test.villain.level.multiplier=1
 
+## Logging configuration
+quarkus.log.level=INFO
```

Comment:
I'm not sure why we need to add `quarkus.log.level=INFO`? That's the default.

Reply:
No but it's handy to remember the property so I can set it to DEBUG in "production" (so I can have more logs when things go wrong)